src/screens/Topics/index.js

```diff
@@ -237,77 +241,35 @@ const styles = StyleSheet.create({
     fontFamily: 'Inter',
     fontStyle: 'normal',
     fontWeight: '400',
-    fontSize: 14,
-    lineHeight: 24,
+    fontSize: normalizeFontSize(14),
+    lineHeight: normalizeFontSize(24),
     color: colors.gray,
     opacity: 0.84,
-    marginTop: 8,
-    marginBottom: 24,
-    paddingHorizontal: 20
+    marginTop: dimen.normalizeDimen(8),
+    marginBottom: dimen.normalizeDimen(24),
+    paddingHorizontal: dimen.normalizeDimen(20)
   },
   containerTopic: {
     flexDirection: 'column',
-    marginBottom: 32
+    marginBottom: dimen.normalizeDimen(32)
   },
   title: {
     fontFamily: 'Inter',
     fontStyle: 'normal',
     fontWeight: 'bold',
-    fontSize: 18,
-    lineHeight: 22,
+    fontSize: normalizeFontSize(18),
+    lineHeight: normalizeFontSize(22),
     color: colors.black,
-    marginBottom: 13,
+    marginBottom: dimen.normalizeDimen(13),
     // textTransform: 'capitalize',
-    paddingHorizontal: 22
+    paddingHorizontal: dimen.normalizeDimen(22)
   },
   listTopic: {
     flexDirection: 'column',
-    marginBottom: 8,
-    paddingRight: 8
+    marginBottom: dimen.normalizeDimen(8),
+    paddingRight: dimen.normalizeDimen(8)
   },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for improving the responsiveness of the application. The use of `normalizeFontSize` function to normalize font sizes is a good practice in React Native to ensure consistent text size across different screen sizes and resolutions.



<!-- This is an auto-generated comment by OSS CodeRabbit -->